src/generate/format-contributor.js

```diff
@@ -5,7 +5,7 @@ const avatarTemplate = _.template(
   '<img src="<%= contributor.avatar_url %>" width="<%= options.imageSize %>px;" alt="<%= name %>"/>',
```

Comment:
[nb] the `width` option here, wonder if we could do height too (as we know github images are squares?). When the table is loading it jumps all over my screen

Reply:
Yeah, we could do that as well.